integration-tests/src/test/java/tachyon/client/table/RawTableIntegrationTest.java

```diff
@@ -39,16 +51,15 @@ public final void after() throws Exception {
   public final void before() throws Exception {
     mLocalTachyonCluster = new LocalTachyonCluster(10000, 1000, Constants.GB);
     mLocalTachyonCluster.start();
-    mTfs = mLocalTachyonCluster.getClient();
+    mTfs = mLocalTachyonCluster.getOldClient();
```

Comment:
Why are we switching from the new client to the old client? That seems backwards.